ovos_core/intent_services/__init__.py

```diff
@@ -68,22 +58,23 @@ def __init__(self, bus):
 
         # TODO - replace with plugins
         self.adapt_service = AdaptService()
-        if PadaciosoService is not PadatiousService:
+        try:
+            from ovos_core.intent_services.padatious_service import PadatiousService
             self.padatious_service = PadatiousService(bus, config['padatious'])
-        else:
-            LOG.error(f'Failed to create padatious handlers, padatious not installed')
+        except ImportError:
+            LOG.error(f'Failed to create padatious intent handlers, padatious not installed')
```

Comment:
Maybe LOG.warning? Error seems too strong here, possibly

Reply:
not sure, we dont have a proper alternative to padatious, it is an error until we get intent pipeline plugins merged, padacioso is a fallback so this could also be argued otherwise... should probably be scary in the logs to make it clear something is wrong, since padacioso is not performant at all. can also tweak the log message instead